Gopkg.toml

```diff
@@ -32,4 +32,8 @@
 [[constraint]]
   name = "github.com/fabric8-services/fabric8-common"
   revision = "2efd83d29b1e5c657728e83fe03878569c0f122f"
-  
\ No newline at end of file
+
+[[constraint]]
```

Comment:
why do you add the repository as a self-dependency?

Reply:
There is no `https://github.com/fabric8-services/fabric8-auth-cli` and that was the only way for me to get it to compile, unless I do something like 



```diff
diff --git a/main.go b/main.go
index a394fdf..04e029e 100644
--- a/main.go
+++ b/main.go
@@ -6,8 +6,8 @@ import (

        "github.com/sirupsen/logrus"

-       "github.com/fabric8-services/fabric8-auth-cli/cmd"
        "github.com/fabric8-services/fabric8-common/log"
+       "github.com/xcoulon/fabric8-auth-cli/cmd"
 )

 func main() {
```

Reply:
ah, ok, so maybe it's time to promote it to the `fabric8-services` org, I believe ;)

Reply:
ah sounds perfect :)